src/lang/en-US.json

```diff
@@ -959,6 +959,36 @@
   "histograms_name": "Histograms",
   "histograms_description": "Percentages indicate win rates for the labeled bin",
 
+  "histograms_kills_description" : "",
```

Comment:
To save space I think if the string is blank then it should not be added to this file.  You can default to empty string.